src/Uno.UWP/Media/SpeechRecognition/SpeechRecognizer.Android.cs

```diff
@@ -0,0 +1,86 @@
+#if __ANDROID__
```

Comment:
this _pragma_ is not required when the file is named `.Android.cs`.

Reply:
Removed